src/OrbitGl/CaptureWindowTest.cpp

```diff
@@ -79,4 +81,193 @@ TEST_F(UnitTestCaptureWindow, SlidersRespondToMouseLeave) {
   EXPECT_FALSE(unit_test_vertical_slider->IsMouseOver());
 }
 
+class CaptureClientAppInterfaceFake : public orbit_capture_client::AppInterface {
+  [[nodiscard]] orbit_capture_client::CaptureClient::State GetCaptureState() const {
+    return orbit_capture_client::CaptureClient::State::kStopped;
+  }
+
+  [[nodiscard]] bool IsCapturing() const { return false; }
+
+  void StartCapture(){};
+  void StopCapture(){};
+  void AbortCapture(){};
+  void ToggleCapture(){};
+};
+
+class NavigationTestCaptureWindow : public CaptureWindow, public testing::Test {
+ public:
+  explicit NavigationTestCaptureWindow() : CaptureWindow(nullptr) {
+    capture_client_app_ = &capture_client_app_fake_;
+
+    Resize(600, 100);
+
+    capture_data_ = TrackTestData::GenerateTestCaptureData();
+    time_graph_ = std::make_unique<TimeGraph>(this, nullptr, &viewport_, capture_data_.get(),
+                                              &picking_manager_);
```

Comment:
Could we use `CreateTimegraph` here?

Reply:
Hm I would rather leave it as-is to clearly indicate the `nullptr` that is passed as `app_`.